### PR TITLE
Unhelpful title in tm_g_association

### PR DESCRIPTION
**Pull request by chlebowa**
Closes [this issue](https://github.com/insightsengineering/teal.modules.general/issues/399).

Fixed the way the title of the plot panel is generated.
Improved check box labels in the module UI.
Modified labels for histogram Y axis (_Proportional_ -> _Fraction_).

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 ### Enhancements
 
 * Added `parent_dataname` argument to `tm_variable_browser` and `tm_missing_data` to allow specification of parent dataset for these modules.
+* Better UI labels in `tm_g_association`.
```
Comment (nikolas-burkoff) [inline NEWS.md:13]:
```suggestion
* Improved UI labels and plot titles in `tm_g_association`.
```

We typically use past tense here if possible

File: R/tm_g_association.R [hunk 2]
```diff
@@ -359,22 +359,45 @@ srv_tm_g_association <- function(id,
         )
       })
 
+      # helper function to format variable name
+      foo <- function(x) {
```
Comment (nikolas-burkoff) [inline R/tm_g_association.R:363]:
```suggestion
      format_labels_helper <- function(x) {
```
(and changes below of course)

Best to be more descriptive than just "foo"

File: R/tm_g_association.R [hunk 3]
```diff
@@ -359,22 +359,45 @@ srv_tm_g_association <- function(id,
...
+            sprintf(
+              "Associations between %s and: %s",
+              ref_cl_lbl,
+              paste(lapply(vars_names, foo), collapse = ", ")
+            )
+          )
+        } else {
+          switch(as.character(length(vars_names)),
+            "0" = sprintf("Value distribution for %s", ref_cl_lbl),
+            sprintf(
```
Comment (nikolas-burkoff) [inline R/tm_g_association.R:388]: So line 97 -99 I think means you are not allowed multiple reference values (i.e. if you put muiltiple = TRUE ln examples on line 48 it won't work so I don't think you need this?

File: R/tm_g_bivariate.R [hunk 4]
```diff
@@ -751,7 +751,7 @@ bivariate_ggplot_call <- function(x_class = c("NULL", "numeric", "integer", "fac
       plot_call <- reduce_plot_call(
         plot_call,
         quote(geom_bar(aes(y = ..prop.., group = 1))),
-        quote(ylab("Proportion"))
+        quote(ylab("Fraction"))
```
Comment (nikolas-burkoff) [inline R/tm_g_bivariate.R:754]: Both fraction and proportion are fine here tbh

**Review by nikolas-burkoff — COMMENTED** [review]
A few small comments but looking good

**Comment by nikolas-burkoff** [comment]
(You'll need to either commit again or run `git commit -m "trigger" --allow-empty"` to trigger the automation pipeline after 
image
auto restyling above

File: R/tm_g_association.R (hunk 3 above)
Comment (chlebowa) [reply R/tm_g_association.R:388]: I'm not sure I follow. This is not for multiple references, but for multiple associated values, which is allowed in the current implementation.

File: R/tm_g_association.R (hunk 2 above)
Comment (chlebowa) [reply R/tm_g_association.R:363]: will do

File: R/tm_g_association.R (hunk 3 above)
Comment (nikolas-burkoff) [reply R/tm_g_association.R:388]: hmm  - maybe I misunderstood, let me look more closely

Comment (nikolas-burkoff) [reply R/tm_g_association.R:388]: yup my bad - ignore this 👍

**Review by nikolas-burkoff — APPROVED** [review]
Yup looks good

**Pull request merged**